docs/reference/src/main/asciidoc/environments.asciidoc

```diff
@@ -92,7 +92,7 @@ Setting the identifier is as simple as adding one context parameter into `web.xm
 
 ==== Tomcat
 
-Tomcat 7 and 8 are supported. Context activation/deactivation and dependency
+Tomcat 9 is supported. Context activation/deactivation and dependency
 injection into Servlets and Filters works out of the box. Injection into Servlet listeners works on
 Tomcat 7.0.50 and newer.
```

Comment:
Looking at this I am thinking we might as well drop the last two lines (as we are long past 7.0.50 and it all works out of the box now). Instead we could maybe declare why is 9+ supported (e.g. alignment with EE 8 and Servlet 4 specs?)